python/ray/tune/tune.py

```diff
@@ -373,6 +373,8 @@ class and registered trainables.
     if not trial_executor or isinstance(trial_executor, RayTrialExecutor):
         _ray_auto_init()
 
+    ray._private.usage.usage_lib.record_library_usage("tune")
```

Comment:
would this double count the `_remote=True` case?

Reply:
Oh good question. @jjyao is `record_library_usage` idempotent, or some kind of counter? Because for jobs it's being called once per call to `ray job submit`